src/hashfile.rs

```diff
@@ -3,43 +3,65 @@ use anyhow::anyhow;
 use std::{
     fs::File,
     io::{BufRead, BufReader},
-    path::Path,
+    path::{Path, PathBuf},
 };
 
 /// Common filenames of hash files.
-const HASH_FILES: [(HashAlg, &str); 24] = [
+const HASH_FILES: [(HashAlg, &str); 30] = [
```

Comment:
Let's just change this to be an array slice so we don't have to keep changing the number.

```suggestion
const HASH_FILES: &[(HashAlg, &str)] = &[
```